gpt_index/vector_stores/redis.py

```diff
@@ -0,0 +1,313 @@
+"""Redis Vector store index.
+
+An index that that is built on top of an existing vector store.
+"""
+import ast
+import logging
+from typing import Any, Dict, List, Optional
+
+import numpy as np
+
+from gpt_index.data_structs.node_v2 import DocumentRelationship, Node
```

Comment:
we finally committed to replacing all `gpt_index` with `llama_index`. 

Sorry for the myriad of refactoring changes!